scripts/versions

```diff
@@ -10,6 +10,6 @@ declare -A VERSIONS=(
     ["runc"]=v1.1.9
     ["crun"]=1.9.2
     ["bats"]=v1.10.0
-    ["buildah"]=v1.28.0
+    ["buildah"]=v1.33.1
```

Comment:
Can we add this to `dependencies.yaml`? One reason that it's so much outdated is that the yaml file is the entry point for regularly checking new versions (manually).

Reply:
Done. PTAL

Reply:
This one has to be changed now as well to v1.33.2